### PR TITLE
Example of use of tools/Jenkinsfile

### PR DESCRIPTION
**Pull request by maht**
Companion pull request for the one at [this pull request](https://github.com/apache/incubator-nuttx-testing/pull/2) to illustrate relation of the two repositories. See description there.

Not intended to be merged right now.

### CODE REVIEW

**Comment by xiaoxiang781216** [comment]
@liuguo09 please take a look.

File: tools/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
```
Comment (patacongo) [inline tools/Jenkinsfile]: Is a license header required?  I think yes.

Comment (maht) [reply tools/Jenkinsfile]: Right, I added it.

Comment (xiaoxiang781216) [reply tools/Jenkinsfile]: @maht do we still need this patch, if we you github action to do precheck:
https://github.com/apache/incubator-nuttx/pull/261

Comment (maht) [reply tools/Jenkinsfile]: @xiaoxiang781216 No, we don't need it anymore. It can be closed.

**Pull request closed without merging**